exercises/finale-workflow/Client/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Temporalio.Client;
+using Temporalio.Finale.Workflow;
+
+var client = await TemporalClient.ConnectAsync(new ()

```

Comment:
Inconsistent with how we are providing the target host to connect compared to #5 and #6